en/docs/samples/saml2-bearer-assertion-profile.md

```diff
@@ -0,0 +1,124 @@
+# Set Up SAML2 Bearer Assertion Profile
+
+This page guides you through exchanging a SAML2 assertion for a valid OAuth access token in order to access a resource in a sample application.
+
+{!fragments/register-a-service-provider.md!}
+
+----
+
+{!fragments/oauth-app-config-basic.md!}
+
+!!! tip
+    -   Make sure the **SAML2** grant is enabled under **Allowed Grant Types** when configuring OAuth/OpenID Connect.
+
+    -   You can provide any valid URL as the **Callback URL**. This URL value is not used for any other operations during this sample.
+
+----
+
+## Set up the sample application
+
+{!fragments/travelocity.md!}
+
+----
+
+## Configure SSO for the sample app
+
+1. Click **List** under **Service Providers**.
+
+2. Click **Edit** to modify the service provider you created for the Travelocity sample. 
+
+3. Modify the **SAML2 Web SSO Configuration** and enable the following fields.
+    - **Enable Audience Restriction**
+    - **Enable Recipient Validation**
+
+4. Enter the following values. 
+    - **Audience**: `https://localhost:9443/oauth2/token`
+    - **Recipient**: `https://localhost:9443/oauth2/token`
+
+    ![enable-audience-restriction](../assets/img/samples/enable-audience-restriction.png) 
+
+5. Click **Update** to save changes. 
+
+-----
+
+    
+## Enable SAML2 grant
+
+1. Open the `travelocity.properties` file found in the `<TOMCAT_HOME>/webapps/travelocity.com/WEB-INF/classes` folder.
+
+2. Edit the following configurations.
+
+    Enter the client ID and client secret you recieved when registering the OAuth/OpenID Connect application.
+
+    ``` java
+    EnableOAuth2SAML2Grant=true
+    OAuth2.ClientId= (enter the client ID received at the application registration)
+    OAuth2.ClientSecret= (enter the client secret received at the application registration)
+    ```
+
+    
+2. Restart the Tomcat server to apply the changes. 
+
+-----
+
+## Try it out
+
+1. Start the Tomcat server and access the following URL on your browser: <http://wso2is.local:8080/travelocity.com>
+
+	```
+	http://<TOMCAT_HOST>:<TOMCAT_PORT>/travelocity.com
+	```
+
+2. Click **Click here to login with SAML from Identity Server (Post binding or Redirect Binding)**. 
+
+    You are redirected to WSO2 Identity Server for authentication. 
+
+3.  Enter the username and password and click **SIGN IN**.  
+
+    ![travelocity-sign-in](../assets/img/samples/travelocity-sign-in.png) 
+
+4.  Click **Request OAuth2 Access Token** to receive the access token.  
+
+    ![request-oauth2-access-token](../assets/img/samples/request-oauth2-access-token.png)  
+
+    You will receive an access token as shown below.
+
+    ![oauth2-token-details](../assets/img/samples/oauth2-token-details.png) 
+
+6.  Use the introspection endpoint of WSO2 Identity Server to get the token information.
+
+    **Request**
```

Comment:
Shall we use tabs for these requests and responses?

Reply:
we have used tabs to give the request format and sample request